canonical-web-and-design/maas-fe
# Add run script.

Note, `yarn.lock` added to `.gitignore` as `./run` erroneously generates a yarn.lock in the yarn workspace.

## QA
1. Pull and run ./run

The app should run and be available from http://localhost:8000
